File: app/src/main/java/com/inguana/weatherapp/model/Area.java
package com.inguana.weatherapp.model;

import com.google.gson.annotations.SerializedName;
import com.inguana.weatherapp.network.networkModel.response.Request;

import java.util.List;

public class Area {

    @SerializedName("weather")
    private List<WeatherDay> weatherDayList;

    @SerializedName("request")
    private List<Request> request;

    //=================================================
    //TODO: implement current condition to the app
    //=================================================

    public Area(List<WeatherDay> weatherDayList, List<Request> request) {
        this.weatherDayList = weatherDayList;
        this.request = request;
    }

    public List<WeatherDay> getWeatherDayList() {
        return weatherDayList;
    }

    public void setWeatherDayList(List<WeatherDay> weatherDayList) {
        this.weatherDayList = weatherDayList;
    }

    public List<Request> getRequest() {
        return request;
    }

    public void setRequest(List<Request> request) {
        this.request = request;
    }

    @Override
    public String toString() {
        return "Data{" +
                "weatherDayList=" + weatherDayList +
                ", request=" + request +
                '}';
    }
}

File: app/src/main/java/com/inguana/weatherapp/model/Area.kt
package com.inguana.weatherapp.model

import com.google.gson.annotations.SerializedName
import com.inguana.weatherapp.network.networkModel.response.Request

class Area     //=================================================
//TODO: implement current condition to the app
//=================================================
    (
    @field:SerializedName("weather") var weatherDayList: List<WeatherDay?>, @field:SerializedName(
    "request"
) var request: MutableList<Request?>
) {

    override fun toString(): String {
        return "Data{" +
                "weatherDayList=" + weatherDayList +
                ", request=" + request +
                '}'
    }
}